### PR TITLE
Remove spark-rapids/tools

### PR DESCRIPTION
**Pull request by GaryShen2008**
Moved to spark-rapids-tools/core

Contribute to https://github.com/NVIDIA/spark-rapids-tools/issues/46

Signed-off-by: Gary Shen <gashen@nvidia.com>

### CODE REVIEW

**Comment by GaryShen2008** [comment]
Need to decide how to deal with the docs.

**Comment by pxLi** [comment]
build

**Review by pxLi — COMMENTED** [review]
LGTM for the CI part

**Review by NvTimLiu — DISMISSED** [review]
LGTM,

I'll update the release scripts later: 

https://github.com/NVIDIA/spark-rapids/blob/branch-23.02/jenkins/deploy.sh#L95

**Comment by GaryShen2008** [comment]
The build script has been updated.
The only remaining question is if we want to keep docs/spark-profiling-tool.md and spark-qualification-tool.md in spark-rapids repo.
Personally, I prefer to move them to spark-rapids-tools repo to keep updated with the code behavior.
What do you think, @mattahrens?

**Comment by mattahrens** [comment]
As discussed on Slack, let's keep the documentation for Github pages centralized with other Spark RAPIDS documentation.

**Comment by GaryShen2008** [comment]
build

File: README.md [hunk 1]
```diff
@@ -56,16 +56,8 @@ access to any of the memory that RMM is holding.
...
-The Qualification tool is used to look at a set of applications to determine if the RAPIDS Accelerator for Apache Spark
-might be a good fit for those applications.
-
-The Profiling tool generates information which can be used for debugging and profiling applications.
-Information such as Spark version, executor information, properties and so on. This runs on either CPU or
-GPU generated event logs.
-
-Please refer to [Qualification tool documentation](docs/spark-qualification-tool.md)
-and [Profiling tool documentation](docs/spark-profiling-tool.md)
-for more details on how to use the tools.
```
Comment (mattahrens) [inline README.md:68]: We want to leave this comment in the core README as we are leaving documentation in the spark-rapids repo.

**Review by mattahrens — APPROVED** [review]
One minor comment related to persist link to markdowns in README but otherwise fine

File: README.md (hunk 1 above)
Comment (GaryShen2008) [reply README.md]: Added it back.

**Comment by GaryShen2008** [comment]
build

**Review by mattahrens — APPROVED** [review]

File: jenkins/spark-nightly-build.sh [hunk 2]
```diff
@@ -120,13 +119,12 @@ distWithReducedPom "install"
 if [[ $SKIP_DEPLOY != 'true' ]]; then
     distWithReducedPom "deploy"
 
-    # this deploy includes 'tools' that is unconditionally built with Spark 3.1.1
+    # this deploy sub modules except dist that is unconditionally built with Spark 3.1.1
```
Comment (gerashegalov) [inline jenkins/spark-nightly-build.sh:122]: minor typo
```sugestion
    # this deploys submodules except dist that is unconditionally built with Spark 3.1.1
```

**Review by gerashegalov — APPROVED** [review]
LGTM

File: jenkins/spark-nightly-build.sh (hunk 2 above)
Comment (GaryShen2008) [reply jenkins/spark-nightly-build.sh:122]: ok, let me file a follow-up PR to fix it.

**Review by zhanga5 — APPROVED** [review]
LGTM

**Pull request merged**